extensions/wikia/SemanticMediaWiki/maintenance/SMW_runRefreshDataUsingMigrationJob.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+
+
+require_once ( getenv( 'MW_INSTALL_PATH' ) !== false
+	? getenv( 'MW_INSTALL_PATH' ) . "/maintenance/commandLine.inc"
+	: dirname( __FILE__ ) . '/../../../../maintenance/commandLine.inc' );
+
+global $wgCityId;
+
+$task = new SMW_MigrationJob();
+(new \Wikia\Tasks\AsyncTaskList())
+	->wikiId($wgCityId)
+	->add($task->call('run', $wgCityId))
```

Comment:
I'm wondering if we actually need to pass city_id in both place (add wikiId task param and the called method itself). Maybe just pick one?